docs/reference/security/authentication/jwt-realm.asciidoc

```diff
@@ -294,7 +294,7 @@ token.
 as `HS256`. The algorithm must be in the realm's allow list.
 
 `typ`::
-(Optional, String) Indicates the token type, which must be `JWT`.
+(Optional, String) Indicates the token type. For an ID token, this must be `JWT`; for access tokens, this must be `JWT` or `at+jwt`.
```

Comment:
Nit: unless its a pain because of asciidoc formatting, lets match the itemized format from: https://github.com/elastic/docs-content/pull/1124/files#diff-3761114bd79abf4acad1c5d598f9cbafc116580eeb95beeda3dc0f1466fdd5fcR221